contracts/FjordDrop.sol

```diff
@@ -34,38 +34,42 @@ contract FjordDrop is Erc721BurningErc20OnMint, ReentrancyGuard, IERC2981 {
     error FJORD_PublicMintisNotActive();
     error FJORD_FjordIsNotActive();
 
-/*//////////////////////////////////////////////////////////////
+    /*//////////////////////////////////////////////////////////////
                         EVENTS
 //////////////////////////////////////////////////////////////*/
 
     event MintedAnNFT(address indexed to, uint256 indexed tokenId);
 
-/*//////////////////////////////////////////////////////////////
+    /*//////////////////////////////////////////////////////////////
                         STATE VARIABLES
 //////////////////////////////////////////////////////////////*/
 
     uint16 public mintCounter;
     uint16 public constant TOTAL_SUPPLY = 525;
     string public customBaseURI;
-    string public contractURI =
-        "ipfs://QmdyYCtUsVsC5ymr7b4txQ6hXHpLXtJU7JXCDuHEJdXnRe";
-    address private mainnetFjordAddress =
-        0x6f435948d9ad4cA0a73a3257743528469899ceec;
+    string public contractURI = "ipfs://QmdyYCtUsVsC5ymr7b4txQ6hXHpLXtJU7JXCDuHEJdXnRe";
+    //TODO - remove mainnetFjordAddress
+    address private mainnetFjordAddress = 0x6f435948d9ad4cA0a73a3257743528469899ceec;
     uint256 public whitelistEndDate;
     uint256 private constant PRICE_PER_WHITELIST_NFT = 0.02 ether;
     bytes32 public whiteListSaleMerkleRoot;
     uint32 private constant MAX_MINT_PER_WHITELIST_WALLET = 2;
     mapping(address => uint32) public mintPerWhitelistedWallet;
+    //TODO - make this a public variable and add getter
+    // minor but should we default to something? otherwise we try to do a calculation on undefined
```

Comment:
suggestion on being able to view currently set value publicly and setting this to a default